RESPONSIBILITIES.md

```diff
@@ -56,6 +56,17 @@ It's our responsibility to ensure the content and code in pull requests are corr
 - If a pull request is valuable but isn't gaining traction, consider reaching out to fulfill the necessary requirements. This way, the pull request can be merged, even if the work is done by several individuals.
 - Lastly, strive for progress, not perfection.
 
+### Merging a Pull Request 
+
+It is important that commit messages are helpful in understanding the reasons for a given commit and maintain good commit hygiene by only keeping the relevant information. 
+
+When a PR is going to be merged, our repositories are set to automatically squash the commits into a single commit. This process needs human intervention to produce high quality commit messages, with the following steps to be followed as much as possible:
+
+- The commit subject is clean and conveys what is being merged.
```

Comment:
These don't read like steps. Maybe "Ensure that the commit subject ... Ensure that ..."?

Reply:
I reworded this as `The commit subject should be concise and clearly convey what is being merged.` right now. Let me know if you think we need to reword further.